### ISSUE TITLE
Can you include intel wifi kext?

### ISSUE THREAD

**Issue by NebulaStride-23798** [MSzturc/Lenovo-T460-OpenCore#14]
First of all please accept my thanks for providing us a well-presented opencore config and I just installed Catalina successfully.

Just have an idea if you can include the intel wireless driver then it can be a more user-friendly opencore config as a "generic" T460 usually comes with an Intel WiFi card installed.  

the intel WiFi Driver is here [https://github.com/OpenIntelWireless/itlwm](https://github.com/OpenIntelWireless/itlwm)
and I have installed and tested working on my T460

Best Regards

**Comment by aptonline** [comment]
I'd be interested to see if Hand-off and continuity work with that kext. Can you confirm?

**Comment by NebulaStride-23798** [comment]
> I'd be interested to see if Hand-off and continuity work with that kext. Can you confirm?

I don’t think so. Just pure WiFi and Bluetooth support (Bluetooth support via another [kext](https://github.com/OpenIntelWireless/IntelBluetoothFirmware)).

**Comment by aptonline** [comment]
So combined they should provide the relevant environment for Hand-off and Continuity to work. Worth testing I think at some point.

**Comment by NebulaStride-23798** [comment]
> So combined they should provide the relevant environment for Hand-off and Continuity to work. Worth testing I think at some point.

I don’t know if that will happen. As their project introduction stated, they are not building the driver on top of Apple’s native WiFi framework, which might be “ the relevant environment for hand-off” needed.

**Comment by aptonline** [comment]
I'm glad I ordered my DW1560 then :)

**Comment by abdullahjanem** [comment]
I have been trying for two days to get the wifi to work with intel kexts, it works in the setup phase but when I boot to macOS it doesn't work (although Bluetooth does work), can u guys help me with this issue? do u guys have a working efi folder for intel wifi cards? my card is intel 7265NGW  ( I tested this card on Catalina on my specture x360 and it works btw) @metro2450 @aptonline @MSzturc
